acceptance/phase_test.go

```diff
@@ -399,3 +400,19 @@ func SBOMExtensions() []string {
 func SBOMComponents() []string {
 	return []string{"lifecycle", "launcher"}
 }
+
+func assertImageOSAndArch(t *testing.T, imageName string, phaseTest *PhaseTest) { //nolint  - these functions are in fact used, i promise
+	inspect, _, err := h.DockerCli(t).ImageInspectWithRaw(context.TODO(), imageName)
+	h.AssertNil(t, err)
+	h.AssertEq(t, inspect.Os, phaseTest.targetDaemon.os)
+	h.AssertEq(t, inspect.Architecture, phaseTest.targetDaemon.arch)
+	fmt.Println("asserted that image has OS:", phaseTest.targetDaemon.os, "and arch: ", phaseTest.targetDaemon.arch)
```

Comment:
Did you mean to leave this in? I'm fine with it staying but wanted to ask.

Reply:
:) ooops!  good catch - yeah agree it's harmless but I'll delete it,   was leftover from some debugging...